tests/unit/processor/timestamp_differ/test_timestamp_differ.py

```diff
@@ -0,0 +1,286 @@
+# pylint: disable=missing-docstring
+
+import pytest
+
+from logprep.processor.base.exceptions import ProcessingWarning
+from tests.unit.processor.base import BaseProcessorTestCase
+
+test_cases = [  # testcase, rule, event, expected
```

Comment:
please add a test for `delete_source_fields`, `overwrite_target_field` and `extend_target_list`

Reply:
Done